build.gradle

```diff
@@ -120,8 +120,12 @@ subprojects {
             testCompile deps.assertj
             testCompile deps.junit
             testCompile deps.mockito
+        }
 
-
+        test {
+            // Set maxParallelForks to a large number and let gradle to force it to a the
+            // max-workers number when needed.
+            maxParallelForks =  12
```

Comment:
i'm curious why the performance gain is not so significant from 1 to 12 forks.
also maybe considering number of processor here could be more flexible:
e.x:  
maxParallelForks = Runtime.runtime.availableProcessors * 4